frontend/src/pages/landing/landing.tsx

```diff
@@ -0,0 +1,35 @@
+import { type FC } from 'react';
+
+import { Link } from '~/libs/components/components.js';
+import { AppRoute } from '~/libs/enums/app-route.enum';
+import { useEffect, useMousePosition, useState } from '~/libs/hooks/hooks.js';
+
+import styles from './landing.module.scss';
+
+const DEEP = 0.05;
+
+const Landing: FC = () => {
+  const { x, y } = useMousePosition();
+  const [backgroundPosition, setBackgroundPosition] = useState('center');
+
+  useEffect(() => {
```

Comment:
You may also use `useMemo` instead of `useEffect` + `useState` - as the only thing you're using is state value

Reply:
done - [cfc8d8b](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/26/commits/cfc8d8b2a3c891fc827ebc759e2ca48916ed6957)